src/scripts/deploy.coffee

```diff
@@ -42,7 +42,7 @@ module.exports = (robot) ->
   # Displays the available environments for an application
   robot.respond DeploysPattern, (msg) ->
     name        = msg.match[2]
-    environment = msg.match[4] || 'production'
+    environment = msg.match[4]
```

Comment:
You can also probably update the above `DeploysPattern` to make environment non-optional.